scripts/variantstore/wdl/GvsUnified.wdl

```diff
@@ -14,7 +14,6 @@ workflow GvsUnified {
         String project_id
 
         Array[String] external_sample_names
-        Boolean samples_are_controls = false
```

Comment:
this was removed because a unified run with this set up would be useless if it was all controls. The filter model would be made with the controls only....

Reply:
although -- how would "is_control" get set during a unified run?  Ie it may be NA12878/GIAB but how would GVS know?

Reply:
but tbh -- the whole is_control thing has always felt a little cumbersome so I might not be understanding this part